src/components/menu/menu-item/menu-item.component.js

```diff
@@ -151,6 +161,8 @@ const MenuItem = ({
         {...elementProps}
         variant={variant}
         role="menuitem"
+        ariaLabel={ariaLabel}
+        ariaHidden={Boolean(children)}
```

Comment:
The component is still visible on the screen, should we be using `ariaHidden`?

Reply:
An ariaHidden will remove the component from the accessibility tree, making the aria-label redundant, I think

Reply:
[good example](https://stackoverflow.com/questions/52839879/can-i-use-aria-label-and-aria-hidden-true-on-the-same-element#:~:text=1%20Answer&text=aria%2Dhidden%20will%20hide%20the,screen%20readers%20and%20braille%20devices.&text=Many%20screen%20readers%20will%20ignore,you%20also%20specify%20a%20role%20.)

Reply:
ariaHidden is so that the aria label and the menu item text aren't both read out. Otherwise a screen reader might go "menu settings settings" if both are set